modules/oe_translation_cdt/modules/oe_translation_cdt_mock/oe_translation_cdt_mock.info.yml

```diff
@@ -0,0 +1,8 @@
+name: OpenEuropa Translation CDT Mock
```

Comment:
I dont see why we should add this now

Reply:
I must have accidentally select it, will remove.